misc/python/materialize/feature_benchmark/scenarios/benchmark_main.py

```diff
@@ -1891,16 +1885,59 @@ def init(self) -> Action:
             for i in range(0, self.n())
         )
 
+        comment = re.compile(r"--.*?\n", re.IGNORECASE)
+        newline = re.compile(r"\n", re.IGNORECASE)
+
+        create_tpch_tables = "\n".join(
+            f"""
+> {newline.sub(" ", comment.sub("", ddl))}
```

Comment:
It took me a long time to come up with the right regex magic for TD to accept these queries. If there's a better way then please let me know.

Reply:
Generally testdrive is not great for programmatically generating, I might have just pasted the queries in directly (if they are not too huge).

Reply:
It's roughly 1000 lines of SQL

Reply:
Ouch, then makes sense to do it this way.